.github/ISSUE_TEMPLATE/01-compatible-certification-request.md

```diff
@@ -2,7 +2,7 @@
 name: Compatible Certification Request
 about: Start a request for a compatible certification 
 title: 'MicroProfile [Version] Compatible Certification Request'
-labels: 'Certification :trophy:'
+labels: 'certification :trophy:'
```

Comment:
```suggestion
labels: 'certification'
```

It looks like the [label on this repository](https://github.com/eclipse/microprofile/labels) is just `certification` without the :trophy: icon.